internal/app/subsystems/api/service/service.go

```diff
@@ -268,10 +268,6 @@ func (s *Service) sendOrPanic(cq chan *bus.CQE[t_api.Request, t_api.Response]) f
 			Error:      err,
 		}
 
-		select {
-		case cq <- cqe:
-		default:
-			panic("response channel must not block")
```

Comment:
I would be inclined to leave the panic, for example, what happens if a send occurs again before the function completes?